src/consensus/merkle.cpp

```diff
@@ -83,3 +85,55 @@ uint256 BlockWitnessMerkleRoot(const CBlock& block, bool* mutated)
     return ComputeMerkleRoot(std::move(leaves), mutated);
 }
 
+uint256 HashTwoTxIDs(const uint256& txid1, const uint256& txid2) {
```

Comment:
In commit "Add GetCoinBaseMerklePath helper" (c230cc1a72dc4e4079cb2f7b05586e6f32e844ab)

May want to declare this static to avoid creating an unused link symbol.